### PR TITLE
Add docs for apsystems binary sensors

### PR DESCRIPTION
**Pull request by CM000n**
## Proposed change

I have added a few new binary_sensors to the Apsystems integration for diagnostics:
https://github.com/home-assistant/core/pull/123045
The documentation should therefore also be adapted accordingly.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/123045
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Introduced support for `binary_sensor` to enhance integration capabilities.
	- Added new operational data points: 
		- "Off grid status"
		- "Short circuit error status (for input)"
		- "Output status"
	- Updated "Inverter status" clarification for better user understanding.

### CODE REVIEW

File: source/_integrations/apsystems.markdown [hunk 1]
```diff
@@ -23,6 +24,9 @@ The following data is provided by the integration:
 - Lifetime production (Per input and in total)
 - Current production (Per input and in total)
 - Today's production (Per input and in total)
+- Off grid status
+- Short circuit error status (For DC 1 and DC 2)
```
Comment (mawoka-myblock) [inline source/_integrations/apsystems.markdown:28]: Rename `DC 1` and `DC 2` to input 1 and input 2 to be consistent

File: source/_integrations/apsystems.markdown [hunk 2]
```diff
@@ -23,6 +24,9 @@ The following data is provided by the integration:
...
+- Output fault status
```
Comment (mawoka-myblock) [inline source/_integrations/apsystems.markdown]:
```suggestion
- Short circuit error status (per input)
- Off grid status
- Output status
```

**Review by mawoka-myblock — CHANGES_REQUESTED** [review]
And order them

EDIT: just look at the suggestion

**Review by mawoka-myblock — APPROVED** [review]

**Comment by CM000n** [comment]
Is there anything we can do here to get the `in-progress` tag being removed?
Do you have any ideas @mawoka-myblock?

**Comment by mawoka-myblock** [comment]
Nope. Not that I know. Just wait or ping Joost, but I won't as he's busy enough. Btw: I'm working on supporting the other ez1 series inverters too, so pls don't work on that, the ;)

**Comment by CM000n** [comment]
> Nope. Not that I know. Just wait or ping Joost, but I won't as he's busy enough. Btw: I'm working on supporting the other ez1 series inverters too, so pls don't work on that, the ;)

All right, I'll hold back 😆

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @CM000n :+1:

**Pull request merged**